sdk/storage/azure-storage-file-share/tests/test_share.py

```diff
@@ -834,6 +834,37 @@ def test_list_shares_with_snapshot(self, **kwargs):
         share.delete_share(delete_snapshots=True)
         self._delete_shares()
 
+    @FileSharePreparer()
+    @recorded_by_proxy
+    def test_delete_snapshots_options(self, **kwargs):
```

Comment:
Was there an old test for this? If so, we should remove it or merge the two since this covers the old boolean as well. Bonus points if you delete the old recording from assets (Will have to manually find/delete it in the asset repo and then push it)